### PR TITLE
Email feature

### PR DESCRIPTION
**Pull request by homeski**

### CODE REVIEW

**Comment by robojukie** [comment]
image

UI - buttons should stay inside the modal

File: lab/lab-web/src/main/webapp/www/components/performances/add-modal/add.html [hunk 1]
```diff
@@ -13,15 +13,15 @@ <h3 class="modal-title">Request a Booking</h3>
...
-
         <label>Performance Type</label>
         <select class="form-control" ng-model="add.selectedType" ng-change="add.setPerformanceType()"
           ng-options="accomodation as accomodation.label for accomodation in add.accomodations"></select>
 
-
         <label>Description</label>
         <textarea class="form-control" ng-model="add.performance.performer.description"></textarea>
 
+        <label>Email</label>
```
Comment (robojukie) [inline lab/lab-web/src/main/webapp/www/components/performances/add-modal/add.html]: Email should probably sit under Performer Name, I think it's better UX

**Review by robojukie — CHANGES_REQUESTED** [review]

**Comment by homeski** [comment]
fixed both comments

**Comment by robojukie** [comment]
On venue add modal email would be better back under name, before city

**Pull request merged**